products/redis/terraform.yaml

```diff
@@ -0,0 +1,67 @@
+# Copyright 2017 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Terraform::Config
+overrides: !ruby/object:Provider::ResourceOverrides
+  Instance: !ruby/object:Provider::Terraform::ResourceOverride
+    id_format: "{{project}}/{{region}}/{{name}}"
+    custom_code: !ruby/object:Provider::Terraform::CustomCode
+      decoder: 'templates/terraform/redis/instance_decoder.erb'
+    properties:
+      authorizedNetwork: !ruby/object:Provider::Terraform::PropertyOverride
+        default: !ruby/object:Provider::Terraform::Default
+          from_api: true
+      locationId: !ruby/object:Provider::Terraform::PropertyOverride
+        default: !ruby/object:Provider::Terraform::Default
+          from_api: true
+      name: !ruby/object:Provider::Terraform::PropertyOverride
+        custom_expand: 'templates/terraform/redis/expand_name.erb'
+        custom_flatten: 'templates/terraform/redis/flatten_name.erb'
+      redisVersion: !ruby/object:Provider::Terraform::PropertyOverride
+        default: !ruby/object:Provider::Terraform::Default
+          from_api: true
+      region: !ruby/object:Provider::Terraform::PropertyOverride
+        required: false
+        default: !ruby/object:Provider::Terraform::Default
+          from_api: true
+      reservedIpRange: !ruby/object:Provider::Terraform::PropertyOverride
```

Comment:
Do we want to see :BASIC as the default value for `tier`? And set required to false